randomPlexMovie.py

```diff
@@ -12,8 +12,8 @@
 # If you get an OSError saying that the Chrome installation cannot be found,
 # change use_custom_path to True and specify the file path leading
```

Comment:
Maybe change the instruction here too. Someone may not know were to change it

Reply:
Was just about to change that, I'm on it!